File: firstcode/firstassignemt/newassignement/mathematicalpie.py
number = 4
pi = 0
terms = 0
for count in range(1, 5):
	if count % 2 == 1:
		terms = number/count
	elif count % 2 == 0:
		terms = -(number/count)
	pi = pi + terms
	print(pi)
	
	
		
		
	
	 
	

File: firstcode/firstassignemt/myjavaassignment/MathematicalPie.java
public class MathematicalPie {
	public static void main(String[] args) {
		System.out.println("Welcome to my math pie calculator");
	
	int number = 4;
	int pi = 0;
	int terms = 1;
	int odd = 1;
	
	for(int counter = 1; counter <= 301; counter++) {
		if(counter % 2 == 1) {
			terms = number/odd;
		}
		else if (counter % 2 == 0) {
			terms = -(number/odd);
}
		pi = pi + terms;
		odd = odd + 1;
	System.out.println(pi);
}	 
}
}